modules/ding_provider/ding_provider.module

```diff
@@ -587,11 +590,16 @@ function ding_provider_get_pseudo_entity($ding_entity_id) {
   $ting_entity->ding_entity_id = $ding_entity_id;
   $ting_entity->provider_id = $local_id;
   $ting_entity->ding_entity_type = 'ding_entity';
-  $ting_entity->reply = new TingClientObject();
+  // TODO BBS-SAL: Clean out the direct property sets when done.
+  $ting_entity->reply = new TingPseudoObject();
```

Comment:
We'll have to punt this, I will refactor a bit to get rid of the "pseudo" name, and then we will have to see if we can eliminate all calls to ding_provider_get_pseudo_entity